### PR TITLE
Lesson 15 wiki notes (The SQL Sequel).

### PR DESCRIPTION
**Pull request by jimflores5**

### CODE REVIEW

File: Class-15-(SQL-Part-2).md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+1. Review one-to-many relationships, and provide some practical, real-life examples.
+    1. The book references one-to-one and many-to-many relationships but does not go into detail. Provide some scenarios and examples for situations that use these relationships.
+1. It would be helpful to give a live-coding example in MySQL Workbench for setting up a foreign key.
+1. The book showed how to place a subquery inside a ``WHERE`` clause. Discuss the other alternatives for placing a subquery inside a SQL ``SELECT`` command. Provide examples!
+    1. When used in a ``FROM`` clause, the subquery must be given an alias.
+1. As time permits, discuss adding subqueries to ``INSERT`` and ``UPDATE`` statements.
+1. Joins and subqueries are both abstract concepts, and some students may see them as accomplishing the same thing. Spend some time discussing:
+    1. The differences between joins and subqueries,
+    1. When to use one approach over the other,
```
Comment (gildedgardenia) [inline Class-15-(SQL-Part-2).md]: These points should end in periods.

File: Class-15-(SQL-Part-2).md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    1. How to use a join and a subquery.
+1. Review the studio!
+
+## Small Group Time: Class 15 Studio (TA Notes)
+
+1. Remember to check in with each of your students. A good time to shoulder-surf today is during the warm-up SQL query tasks.
+1. Just like SQL part 1, the students will need to download and run a canned set of SQL instructions to create and populate the necessary tables. Some may struggle with this, so be ready to help them through the process.
```
Comment (gildedgardenia) [inline Class-15-(SQL-Part-2).md]: I think this point should be rephrased to say students will need to "make a new model, create tables, and download and import data".

**Review by gildedgardenia — APPROVED** [review]
Looks great!

File: Class-15-(SQL-Part-2).md (hunk 1 above)
Comment (jimflores5) [reply Class-15-(SQL-Part-2).md]: Fixed.  Thanks.

File: Class-15-(SQL-Part-2).md (hunk 2 above)
Comment (jimflores5) [reply Class-15-(SQL-Part-2).md]: Yep.  Your version sounds much better.

Fixed.

**Pull request merged**